pkg/pointerdb/pdbclient/client_test.go

```diff
@@ -14,22 +14,22 @@ import (
 	"github.com/golang/mock/gomock"
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes"
+	"github.com/grpc-ecosystem/go-grpc-middleware/util/metautils"
```

Comment:
I don't think adding this package is essential.

Let's try avoid pulling large dependencies if the implementation is doable without significant effort. (For prototyping it's obviously fine).

Reply:
I will remove it.